helix-view/src/editor.rs

```diff
@@ -253,6 +256,11 @@ impl Editor {
 
             let id = self.documents.insert(doc);
             self.documents[id].id = id;
+            self.watcher
```

Comment:
Now that I think about it, this needs to deal with renaming too, `:w somename.txt` could set a name for an unnamed file or write an existing file to a new location